thefuck/rules/go_unknown_command.py

```diff
@@ -0,0 +1,18 @@
+from thefuck.utils import get_closest, replace_argument
+
+_GOLANG_COMMANDS = (
```

Comment:
Can you please make that list dynamic, like in https://github.com/nvbn/thefuck/blob/master/thefuck/rules/gem_unknown_command.py#L17 ?